README.md

```diff
@@ -1 +1,301 @@
+  init                                Init build-harness
+  jenkins/run-job-with-tag            Run a Jenkins Job with $(TAG)
+  lint                                Lint terraform code
+  make/lint                           Lint all makefiles
+  packages/delete                     Delete packages
+  packages/install                    Install packages 
+  packages/install/%                  Install package (e.g. helm, helmfile, kubectl)
+  packages/reinstall                  Reinstall packages
+  packages/uninstall/%                Uninstall package (e.g. helm, helmfile, kubectl)
+  readme                              Alias for readme/build
+  readme/build                        Create README.md by building it from README.yaml
+  readme/init                         Create basic minimalistic .README.md template file
+  readme/lint                         Verify the `README.md` is up to date
+  semver/export                       Export semver vars
+  semver/show                         Show
+  stages/export                       Export stages vars
+  template/build                      Create $OUT file by building it from $IN template file
+  template/deps                       Install dependencies
+  terraform/get-modules               Ensure all modules can be fetched
+  terraform/get-plugins               Ensure all plugins can be fetched
+  terraform/install                   Install terraform
+  terraform/lint                      Lint check Terraform
+  terraform/validate                  Basic terraform sanity check
+  travis/docker-login                 Login into docker hub
+  travis/docker-tag-and-push          Tag & Push according Travis environment variables
+  verify_gateway_email                Verify $EMAIL used as gateway for forwarding
+
+```
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| attributes | Additional attributes (e.g. `1`) | list | `<list>` | no |
+| delimiter | Delimiter to be used between `namespace`, `stage`, `name` and `attributes` | string | `-` | no |
+| domain | Root domain name | string | - | yes |
+| forward_emails | Emails forward map | map | `<map>` | no |
+| name | Application or solution name (e.g. `app`) | string | `ses` | no |
+| namespace | Namespace (e.g. `cp` or `cloudposse`) | string | - | yes |
+| region | AWS Region the SES should reside in | string | `us-west-2` | no |
+| relay_email | Email that used to relay from | string | - | yes |
+| spf | DNS SPF record value | string | `v=spf1 include:amazonses.com -all` | no |
+| stage | Stage (e.g. `prod`, `dev`, `staging`) | string | - | yes |
+| tags | Additional tags (e.g. map(`BusinessUnit`,`XYZ`) | map | `<map>` | no |
+
+
+
+
+## Related Projects
+
+Check out these related projects.
+
+- [fluentd-datadog-logs](https://github.com/cloudposse/fluentd-datadog-logs) - Dockerfile that implements fluentd log forwarding to datadog's new log service
```

Comment:
Why do you think these projects related?

Reply:
due to label. It isn't related?